app/src/main/res/layout/component_collection_creation.xml

```diff
@@ -10,11 +10,15 @@
     android:layout_height="match_parent"
     android:clipToPadding="false">
 
+    <!-- RtlHardcoded ignored because we want to keep left/right padding regardless of layout direction -->
```

Comment:
So when this is in RTL - how does it look? And why not use start/end?

Reply:
I have changed this. I was thinking the back button will always be in the left, but we also reverse the position, so we still need smaller padding at the "end". TY for catching this!